db/sql.h

```diff
@@ -141,6 +141,7 @@ typedef struct osqlstate {
     /* == sql_thread == */
     char *host;              /* matching remote node */
     unsigned long long rqid; /* per node offload request session */
+    unsigned long long rmtqid; /* per node offload request session */
```

Comment:
Why are we adding a new field here? Can we use rqid?

Reply:
When it's uuid-based, we set rqid to OSQL_RQID_USE_UUID that tells the system to use uuids instead.  So I wanted a different id for remote requests for compatibility with R6.